clippy_utils/README.md

```diff
@@ -0,0 +1,38 @@
+# `clippy-utils`
+
+<!-- begin autogenerated nightly -->
+```
+nightly-2024-11-14
+```
+<!-- end autogenerated nightly -->
```

Comment:
(Nit) This could be moved to be under the "Usage" section, where it is referred to. (As long as the generation script can handle that!)

Reply:
I could, I put it here to put it as early as possible in the `README.md`, so that scrolling isn't necessary to get to the version. I can change this.

Reply:
Moved it.